ui-admin/src/user/DeleteUserModal.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+import { Store } from 'react-notifications-component'
+import Api, { AdminUser, Portal } from 'api/api'
+import { failureNotification, successNotification } from 'util/notifications'
+
+/** Modal to remove a user from a portal. */
```

Comment:
You call this component "modal", but it renders inline.  I think the UX would be better if it were a modal.  Right now the entire table layout has to rearrange to make space for this to render inline, which is jarring.

  See `AdHocEmailModal.tsx` for a good example modal component.

Reply:
Will change.
Sorry, I should have investigated the other modal dialogs more closely.